geode-redis/src/integrationTest/java/org/apache/geode/redis/internal/executor/pubsub/AbstractPubSubIntegrationTest.java

```diff
@@ -51,6 +51,7 @@
 import org.apache.geode.test.junit.rules.ExecutorServiceRule;
 
 public abstract class AbstractPubSubIntegrationTest implements RedisPortSupplier {
+  /* Trigger a stress test run */
```

Comment:
was this intended to be left in?

Reply:
Thanks for pointing that out - I'll remove it.